ansible/roles/smartos-vm/tasks/main.yml

```diff
@@ -18,3 +18,8 @@
 
 - name: Ensure that virtual machine is running
   vmadm: uuid={{ zone_uuid }} state=started
+
+- name: Disable fsync for the virtual machine
+  shell: zfs set sync=disabled "$(vmadm get {{ zone_uuid }} | json {% if zone_brand == 'kvm' %}disks.0.{% endif %}zfs_filesystem)"
```

Comment:
@YanChii Just to be sure: this is not going to be propagated to zfs dumps of our appliances?

Reply:
I'd like to say absolutly not but the truth is that I don't know :).
I've added new commit that returns this setting to normal before snapshotting.
Just to be sure.